src/main/java/nextstep/subway/line/application/LineService.java

```diff
@@ -33,39 +33,30 @@ public LineResponse saveLine(LineRequest request) {
         Station upStation = stationService.findById(request.getUpStationId());
         Station downStation = stationService.findById(request.getDownStationId());
         Line persistLine = lineRepository.save(new Line(request.getName(), request.getColor(), upStation, downStation, request.getDistance()));
-        List<StationResponse> stations = getStations(persistLine).stream()
-                .map(it -> StationResponse.of(it))
+        List<StationResponse> stations = persistLine.getStations().stream()
+                .map(StationResponse::of)
                 .collect(Collectors.toList());
         return LineResponse.of(persistLine, stations);
     }
```

Comment:
StationResponse의 리스트를 생성하는 부분을 LineResponse 내부로 캡슐화해보면 어떨까요 ?
```suggestion
        return LineResponse.of(persistLine);
    }
```